test/functional/repositories_filesystem_controller_test.rb

```diff
@@ -76,14 +79,18 @@ def test_show_no_extension
                  :attributes => { :class => 'line-num' },
                  :sibling => { :tag => 'td', :content => /TEST CAT/ }
     end
+  end
 
-    def test_entry_download_no_extension
+  def test_entry_download_no_extension
+    with_existing_filesystem_scm do |repo_path|
```

Comment:
Unused block argument - `repo_path`. You can omit the argument if you don't care about it.